dataflow/src/main/java/org/checkerframework/dataflow/analysis/Analysis.java

```diff
@@ -132,26 +124,31 @@ public void setCurrentTree(Tree currentTree) {
     /**
      * Construct an object that can perform a org.checkerframework.dataflow analysis over a control
      * flow graph. The transfer function is set later using {@code setTransferFunction}.
+     *
+     * @param maxCountBeforeWidening number of times a block can be analyzed before widening
      */
-    public Analysis(ProcessingEnvironment env) {
-        this(null, -1, env);
+    public Analysis(int maxCountBeforeWidening) {
```

Comment:
I think this should be marked as deprecated and then removed rather than removed in one release.

Reply:
These are used by ErrorProne, so it's important to document this is the change log.
https://github.com/google/error-prone/blob/master/check_api/src/main/java/com/google/errorprone/dataflow/DataFlow.java#L84